mloop_interface.py

```diff
@@ -22,10 +22,13 @@ def set_globals_mloop(mloop_session=None, mloop_iteration=None):
 
 class LoopInterface(Interface):
     def __init__(self):
-        super(LoopInterface, self).__init__()
-
         # Retrieve configuration from file or generate from defaults
         self.config = mloop_config.get()
+
+        # Pass config arguments to parent class's __init__() so that any logging
+        # options are applied appropriately.
+        super(LoopInterface, self).__init__(**self.config)

```

Comment:
Since M-LOOP logging options would only ever belong in the `MLOOP` section of the config file, it would make sense to pass only that section to `LoopInterface.__init__`, i.e.
```python
super(LoopInterface, self).__init__(**self.config['MLOOP'])
```

Reply:
This spurred me to consider a hierarchical configuration, which would warrant ditching INI & configparser in favour of e.g. YAML & PyYAML. But that's another issue.

Reply:
I tried out `**self.config['MLOOP']` but got a `KeyError`. The code here https://github.com/rpanderson/analysislib-mloop/blob/cfc4029535ca6512a35bab4b499875fcfe63acec/mloop_config.py#L71-L78 flattens `config` when making the dictionary that `mloop_config.get()` returns. The arguments could be filtered the way you mentioned but that would require some changes to the config parsing and handling.

Reply:
Ah, thanks for reminding me that the configuration gets flattened. 

> The arguments could be filtered the way you mentioned but that would require some changes to the config parsing and handling.

I agree, and this warrants a separate PR. In lieu of such changes, using the entire parameter dictionary is fine. Resolving.